git-config/src/file/git_config.rs

```diff
@@ -1673,6 +1704,125 @@ mod from_paths {
         assert_eq!(config.len(), 1);
     }
 
+    #[test]
```

Comment:
While tests are failing and that is expected, when pushing you can `#[ignore]` them.

That way CI stays green while you are working on it.